rpc/src/v1/impls/eth.rs

```diff
@@ -49,10 +52,20 @@ impl Eth for EthClient {
 		}
 	}
 
-	// TODO: do no hardcode default sync status
 	fn syncing(&self, params: Params) -> Result<Value, Error> {
```

Comment:
looks like pretty easy to test it
there is a client spawn function in `tests::helpers`